en/docs/learn/forced-password-reset.md

```diff
@@ -163,18 +163,31 @@ template for this option can be configured in the
 
 ### Using SCIM 2.0 API
 
-1. Login to the Management Console.
-2. Add a new external claim by following `main` -> `Claims` -> `Add` ->
-   `Add External Claim`.
-3. Use following values and add.
+1. Open the `scim2-schema-extension.config` file located in the `<IS_HOME>/repository/conf/` folder.
+2. Define the extension by adding the attributes in the following format before the last element of the JSON array in the `scim2-schema-extension.config` file.
+	
+	```xml 
+	{ "attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:adminForcedPasswordReset", "attributeName":"adminForcedPasswordReset", "dataType":"boolean", "multiValued":"false", "description":"Enable password change required notification in the user creation.", "required":"false", "caseExact":"false", "mutability":"readwrite", "returned":"default", "uniqueness":"none", "subAttributes":"null", "canonicalValues":[], "referenceTypes":[] },
+	```
+3. Add the attribute names to the scim2-schema-extension.config file as `subAttributes` of the wso2Extension attribute as shown below.
+	
+	```xml
+	{ "attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User", "attributeName":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User", "dataType":"complex", "multiValued":"false", "description":"Enterprise User", "required":"false", "caseExact":"false", "mutability":"readWrite", "returned":"default", "uniqueness":"none", "subAttributes":"verifyEmail askPassword employeeNumber costCenter organization division department manager adminForcedPasswordReset", "canonicalValues":[], "referenceTypes":["external"] }
+	```
+
+4. Restart the server
+5. Login to the Management Console.
+6. Add a new external claim by following `main` -> `Claims` -> `Add` ->
+   `Add External Claim`. 
```

Comment:
check formatting...

Add a new external claim by navigating to **Main > Claims>...**